userconnhandler_tcp.go

```diff
@@ -0,0 +1,91 @@
+package freki
+
+import (
+	"fmt"
+	"net"
+	"runtime/debug"
+
+	"github.com/pkg/errors"
+)
+
+type UserConnServer struct {
+	port      uint
+	processor *Processor
+	log       Logger
+	listener  net.Listener
+}
+
+func NewUserConnServer(port uint) *UserConnServer {
```

Comment:
My plan is to move these servers to be private rather than exposed to the client code. v2...